recipes/amazon.rb

```diff
@@ -0,0 +1,35 @@
+#
+# Cookbook Name:: osquery
+# Recipe:: centos
+#
+# Copyright 2016, Jack Naglieri
+#
+
+osquery_install node['osquery']['version'] do
+  action  :install_amazon
+  upgrade node['osquery']['repo']['package_upgrade']
+end
+
+osquery_syslog node['osquery']['syslog']['filename'] do
+  action      :create
+  pipe_filter node['osquery']['syslog']['pipe_filter']
+  pipe_path   node['osquery']['options']['syslog_pipe_path']
+  only_if     { node['osquery']['syslog']['enabled'] }
+  notifies    :restart, "service[#{osquery_daemon}]"
+end
+
+osquery_conf osquery_config_path do
+  action      :create
+  schedule    node['osquery']['schedule']
+  packs       node['osquery']['packs']
+  fim_paths   node['osquery']['file_paths']
+  fim_exclude_paths node['osquery']['exclude_paths']
+  pack_source node['osquery']['pack_source']
+  decorators  node['osquery']['decorators']
+  notifies :restart, "service[#{osquery_daemon}]"
+end
+
+service osquery_daemon do
+  action [:enable, :start]
+  provider Chef::Provider::Service::Systemd
```

Comment:
Amazon linux 1 doesn't have systemd - does this work as expected?

Reply:
I figured this out running my integration tests. I did remove this and will repush.

Reply:
Once support for Amazon Linux 2 is added, this will be added back with an if statement for the version